ako-operator/controllers/suite_test.go

```diff
@@ -65,6 +77,7 @@ func getTestDefaultAKOConfig() akov1alpha1.AKOConfig {
 			ServiceType:          akov1alpha1.ServiceTypeStr("ClusterIP"),
 			PassthroughShardSize: akov1alpha1.PassthroughVSSize("SMALL"),
 			NoPGForSNI:           false,
+			EnableMCI:            false,
```

Comment:
@DixitAakash: Don't include `EnableMCI`

Reply:
Removed from AKOConfig CRD Spec but added to ako configmap.